.github/workflows/build.yml

```
@@ -86,6 +86,13 @@ jobs:
               echo "${TAG}"
           done
           echo "alias_tags=${alias_tags[*]}" >> $GITHUB_OUTPUT
+          
+      - name: Get Current Fedora Version
+        id: labels
+        run: |
+          docker pull quay.io/fedora-ostree-desktops/${{ env.SOURCE_IMAGE }}:${{ matrix.major_version }}
+          ver=$(docker inspect quay.io/fedora-ostree-desktops/${{ env.SOURCE_IMAGE }}:${{ matrix.major_version }} | jq -r '.[].Config.Labels["org.opencontainers.image.version"]')
```

Your image is derived from ublue-os not fedora-ostree-desktops so we need to tweak the pull URL to mirror that

```suggestion
          docker pull ghcr.io/ublue-os/silverblue-${{ matrix.image_flavor }}:${{ matrix.major_version }}
          ver=$(docker inspect ghcr.io/ublue-os/silverblue-${{ matrix.image_flavor }}:${{ matrix.major_version }} | jq -r '.[].Config.Labels["org.opencontainers.image.version"]')
```